plugin-cgroupv2/src/k8s/utils.rs

```diff
@@ -134,8 +136,12 @@ pub fn list_all_k8s_pods_file(
     }
 
     for prefix in all_sub_dir {
-        let mut result_vec =
-            kubernetes_list_metric_file_in_dir(&prefix.to_owned(), hostname.clone(), kubernetes_api_url.clone())?;
+        let mut result_vec = kubernetes_list_metric_file_in_dir(
```

Comment:
Why is that function taking owned Strings by the way?

Reply:
I have no idea 🤷‍♂️
@AngeCyp ?

Reply:
Because I'm a neophyte 🤓. This should be corrected in the new version just pushed